components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/introspection/IntrospectionResponse.java

```diff
@@ -91,6 +91,11 @@ public final class IntrospectionResponse {
 
     public static final String AUT = "aut";
 
+    // OPTIONAL
+    // Step-up authentication claims
```

Comment:
```suggestion
    // Optional Step-up authentication claims
```
and let's remove the 94th line

Reply:
Kept unchanged [165dd85](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2788/commits/165dd85530b7ee45fabee45a6dac967caed27fda)